app/views/hosts/edit.html.erb

```diff
@@ -127,6 +127,7 @@
             uib-typeahead="city as city.name for city in getCityLocation($viewValue, current_country.id)"
             placeholder="<%= t '.city_placeholder' %>"
             required>
+            <div class="text-danger" ng-hide="current_city_exist">אם אינך מוצא את עירך ברשימה, הזן עיר אקראית ופנה אלינו לתמיכה.</div>
```

Comment:
1. please use translations
2. change to `בחר עיר אקראית`, since it won't allow for cities not found on the list